### ISSUE TITLE
Desktop Start Prompt Not Suitable For Visually Impaired

### ISSUE THREAD

**Issue by GadgetSteve** [distribution/distribution#2956]
On at least Windows the Docker Desktop start-up prompt, (which comes up after windows start-up), while suitable for screen readers is not operable by keyboard as the buttons do not take focus and lack keyboard shortcuts. This can be a problem for visually impaired people as mouse operations can be an issue. 

Please consider at least one of:

- Using a dialogue which allows focus on the buttons with the default being Yes and tab moving to No.
 - Binding and Displaying some keyboard shortcuts in the dialogue
 - Allowing docker desktop to be set to always run without prompting on startup.

**Comment by milosgajdos** [comment]
Closing. This is the wrong repository to report this in. Please open an issue in https://github.com/docker/roadmap/
